Fix teapot conversion on vents/doors

realthecaffiend opened the pull request:
Have the archo conversion check for vents and transfer over heat/use power settings from the previous object (to prevent things like fridges warming) and door held open status (while I'm in here anyhow)

KentHaeger commented:
Similar feature has been implemented, slightly differently. Thanks for bringing this to my attention!

realthecaffiend committed:
Fix teapot conversion on vents/doors

Have the archo conversion check for vents and transfer over heat/use power settings from the previous object (to prevent things like fridges warming) and door held open status (while I'm in here anyhow)
